### PR TITLE
Legals-Page

### PR DESCRIPTION
**Pull request by mustiwebp**
As the title of this PR implies, the legals page has been added

### CODE REVIEW

File: lib/l10n/app_de.arb [hunk 1]
```diff
@@ -155,5 +155,14 @@
     "password": "Passwort",
     "questionaireDoctorVisit1": "Waren Sie ",
     "questionaireDoctorVisit2": " bei einem Arzt oder in einem Krankenhaus?",
-    "questionaireDoctorVisitUnscheduled": "ungeplant"
+    "questionaireDoctorVisitUnscheduled": "ungeplant",
+    "information5tmg": "Angaben gem. § 5 TMG",
+    "projectDISTANCE": "Das Projekt DISTANCE ist einer der vom Bundesministerium für Bildung und Forschung (BMBF) geförderten \"Digitalen FortschrittsHubs: Gesundheit im Rahmen der Medizin-Informatikinitiative\".",
```
Comment (S1rDan) [inline lib/l10n/app_de.arb:160]: Is it correct that "FortschrittsHubs" is written together?

File: lib/routes.dart [hunk 2]
```diff
@@ -68,6 +69,7 @@ class Routes {
           const ViewTherapyScreen(),
       '/my-therapy-screen/my-therapy': (BuildContext ctx) =>
           const MyTherapiesScreen(),
+      'legals': (BuildContext ctx) => const LegalsPage(),
```
Comment (S1rDan) [inline lib/routes.dart:72]: You should stick to our convention. If you wan't to change it, we can discuss it if you want...
I've opened a new issue on that regard: #67

File: lib/screens/legals.dart [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+*    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:picos/widgets/picos_body.dart';
+import 'package:picos/widgets/picos_screen_frame.dart';
+
+/// Shows the Legal Page of the Picos App.
+class LegalsPage extends StatelessWidget {
```
Comment (S1rDan) [inline lib/screens/legals.dart:24]:
```suggestion
class LegalsScreen extends StatelessWidget {
```
We call everyting screen... naming.

**Review by S1rDan — CHANGES_REQUESTED** [review]
It's fine, just please rename the class and file to meet our convention.
I've opened an issue about the routing. We can approach this sometime later.

File: lib/l10n/app_de.arb (hunk 1 above)
Comment (mustiwebp) [reply lib/l10n/app_de.arb:160]: I received this piece of text from other stakeholders and I suppose that it's written in the way above.

**Review by S1rDan — APPROVED** [review]

**Pull request merged**